nix_fast_build/__init__.py

```diff
@@ -761,9 +762,17 @@ async def run_evaluation(
         )
         if error:
             continue
-        is_cached = job.get("isCached", False)
-        if is_cached:
+        cache_status = job.get("cacheStatus")
+        if cache_status is None:
+            # Legacy attribute
+            if job.get("isCached", False):
+                continue
+        # Skip remotely cached jobs, but still consider
+        # them for pushing if they are cached locally
+        elif cache_status == "cached":
             continue
+        elif cache_status == "local":
+            upload_queue.put_nowait(Build(attr, job["drvPath"], job.get("outputs", {})))
```

Comment:
cc @ipetkov how do you like this?

Reply:
This will save the extra `nix build` command.

Reply:
Works great, thanks!